src/Colog/Polysemy.hs

```diff
@@ -1,7 +1,11 @@
 {- |
-Copyright:  (c) 2019-2020 Kowainik
-SPDX-License-Identifier: MPL-2.0
-Maintainer: Kowainik <xrom.xkov@gmail.com>
+Module                  : Colog.Polysemy
+Copyright               : (c) 2019-2020 Kowainik
+                          (c) 2021 Co-Log
```

Comment:
I found out that copyright on multiple lines looks better than on a single comma-separated line

![image](https://user-images.githubusercontent.com/4276606/139720260-5085401b-39b9-49db-8d28-8eb24c135915.png)